### ISSUE TITLE
Add 'Open With' option to the right-click context menu

### ISSUE THREAD

**Issue by Tetley22** [madskristensen/WorkspaceFiles#32]
I would like the right-click context menu to have an 'Open With' option, so I can open files with a non-default application (for example, I would like to open a .cs file in Notepad++, rather than the default app which is VS).

**Describe the solution you'd like**

Add an 'Open With' option to the context menu, where i can use a selector and choose the desired app.

**Comment by jurakovic** [comment]
I agree with this. We can now open file just with double click (or enter), but that opens only default editor for that file type.
It would be much better experience if we could get the same "Open With" dialog that already exists in VS.

For example, I don't want .editorconfig file to be open in .editorconfig "editor". I want to open it as plan text file.
Example from Solution Explorer/Folder View:
![2024-12-18_133940](https://github.com/user-attachments/assets/d22064ad-65d2-4225-aa29-5fb37c7a51ee)
![image](https://github.com/user-attachments/assets/92a931d9-3bf1-4eff-9c4c-772f6052df30)
